[BUG]: PCSX2 no longer building in linux

### Describe the Bug

PCSX2 no longer builds on linux.  Build log includes the following:
```
CMake Error at cmake/Pcsx2Utils.cmake:16 (message):
  Unsupported platform.
Call Stack (most recent call first):
  CMakeLists.txt:33 (detect_operating_system)
```

### Reproduction Steps

Attempt building

### Expected Behavior

PCSX2 builds properly.

### PCSX2 Revision

51ceab1f3c226508d0745ccf2e8894efb69cdaaa

### Operating System

Linux (64bit) - Specify distro below

### If Linux - Specify Distro

Linux Mint 21.2

### Logs & Dumps

[build.log](https://github.com/PCSX2/pcsx2/files/13914727/build.log)

--- a/cmake/Pcsx2Utils.cmake
+++ b/cmake/Pcsx2Utils.cmake
@@ -1,7 +1,12 @@
 function(detect_operating_system)
+	message(STATUS "CMake Version: ${CMAKE_VERSION}")
+	message(STATUS "CMake System Name: ${CMAKE_SYSTEM_NAME}")
+
 	# LINUX wasn't added until CMake 3.25.
 	if (CMAKE_VERSION VERSION_LESS 3.25.0 AND CMAKE_SYSTEM_NAME MATCHES "Linux")
+		# Have to make it visible in this scope as well for below.
 		set(LINUX TRUE PARENT_SCOPE)
+		set(LINUX TRUE)
 	endif()
 
 	if(WIN32)

